map_server:
  ros__parameters:
    use_sim_time: false
    yaml_filename: "/home/rainier/ros2_ws/src/littleslam_ros2/scripts/map.yaml"
    topic_name: "map"
    frame_id: "map"
